### ISSUE TITLE
VSH: The system storage cannot be accessed (80029C41)

### ISSUE THREAD

**Issue by brian218** [RPCS3/rpcs3#12889]
### Quick summary

When launching the VSH/XMB of S/GEX firmware (version 4.89 and 4.70 respectively), the error `The system storage cannot be accessed (80029C41)` (see screenshot 1) always occurs, and if you press Back, the VSH/XMB will get stuck with something keeping circling at the upper-right corner (see screenshot 2).
**However, the VSH/XMB boots successfully and smoothly with DEX and DECR firmware (version 4.81), so I don't think version or firmware type is the cause.**

In 4.89 Shop:
![screenshot_2022_10_27_23_06_20](https://user-images.githubusercontent.com/17809637/198335521-290ee9b1-7c3e-41b4-a52b-6694f68296a2.png)
![screenshot_2022_10_27_23_06_29](https://user-images.githubusercontent.com/17809637/198335550-52633012-9381-4be4-a935-57f827f1df5f.png)

In 4.70 GEX:
![screenshot_2022_10_27_23_15_57](https://user-images.githubusercontent.com/17809637/198336109-321082d8-5eee-410c-acb1-294eb7dc147d.png)

### Details

Additional information: the exact same error `The system storage cannot be accessed (80029C41)` happens when you select `Settings -> System Update ->Update via Storage Media` in the VSH/XMB of CEX/DEX/DECR firmware.
So it seems like firmware that enforces local firmware update check on boot will run into this error.

In 4.89 CEX:
![screenshot_2022_10_27_23_39_26](https://user-images.githubusercontent.com/17809637/198335627-211f836d-b849-4de7-b75d-381e00053f2e.png)

### Attach a log file

4.89 Shop with v0.0.24-14337-5210df68:
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9880861/RPCS3.log.gz)

### System configuration

11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz | 8 Threads | 15.70 GiB RAM | TSC: 2.803GHz | AVX-512+ | FMA3
Operating system: Windows, Major: 10, Minor: 0, Build: 22000, Service Pack: none, Compatibility mode: 0
GPU: 'Intel(R) Iris(R) Xe Graphics' running on driver 0.402.713

**Comment by brian218** [comment]
Also, when you press Back after the error occurs (on CEX/Shop/DEX/DECR), the system will attempt to delete `/dev_hdd1/PS3UPDATE/ps3version.txt` file and `/dev_hdd1/PS3UPDATE/` folder, resulting in the following errors:
```
SYS: 'sys_fs_disk_free' failed with 0x80010002 : CELL_EINVAL, “/dev_hdd1”
SYS: 'sys_fs_open' failed with 0x8001003a : CELL_ENOTMOUNTED, “/dev_hdd1”
SYS: 'sys_fs_unlink' failed with 0x8001003a : CELL_ENOTMOUNTED, “/dev_hdd1/PS3UPDATE/ps3version.txt”
SYS: 'sys_fs_rmdir' failed with 0x8001003a : CELL_ENOTMOUNTED, “/dev_hdd1/PS3UPDATE”
```
PR #12880 indeed solves the errors listed above, but `The system storage cannot be accessed (80029C41)` still persists, which is the reason why I decide to open an issue.

**Comment by brian218** [comment]
Solved by #12893.
